website/blog/2024-04-15-0.74-post.md

```diff
@@ -0,0 +1,210 @@
+---
+title: 'React Native 0.74'
+authors:
+  [hurali, alanjhughes, alfonsocj, ingridwang, cortinico, huntie]
+tags: [release]
+date: 2024-04-15
+---
+
+Today we're releasing React Native 0.74!...
+
+## Notable
+
+<!-- https://github.com/facebook/react-native/blob/main/CHANGELOG.md -->
+
+- React Native Frameworks
+- Yoga 3.0
+- Batched updates for `onLayout` (New Architecture only?)
+- Preparing for the New Architecture?
+- Deprecated APIs in `PushNotificationIOS`
+- Removed [deprecated prop types](https://github.com/facebook/react-native/commit/228cb80af9ded20107f3c7a30ffe00e24471bfeb)
+- Android SDK minimum bump (Android 6.0)
```

Comment:
@huntie @NickGerleman we need to add a bullet that XCode min support version is 15.x

Reply:
We still test with 14.3 as well, but there was some question of whether we wanted to set min to 15 because folks were no longer able to debug older XCode versions locally.